auth-web/tests/unit/test-utils/test-data/transaction.ts

```diff
@@ -6,7 +6,7 @@ export const transactionResponse: TransactionListResponse = {
     {
       businessIdentifier: '123',
       createdName: 'tester 123',
-      createdOn: new Date('2023-01-24T23:09:46+00:00'),
+      createdOn: new Date('2023-01-24T14:00:00'), // If we don't specify a timezone, it converts to PST.
```

Comment:
I think you should update this comment to: `The api does not provide the timezone information so the date display will set it to a UTC timezone`